brainstormforce/ultimate-addons-for-gutenberg
# Fix: Typography - Cursor should be center issue.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/0ce4e760-9da1-4bd3-a275-1b6d44711c24?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- Output - https://share.getcloudapp.com/geum2NPO

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop any block.
- Check typography setting check Cursor is in the center or not.

### Checklist:
